tests/test_lab_serve.py

```diff
@@ -11,10 +12,13 @@
 
 # First Party
 from instructlab import configuration, lab
+from instructlab.model.backends.backends import get_max_stable_vram_wait
 from instructlab.model.serve import warn_for_unsuported_backend_param
 
 CFG_FILE_NAME = "test-serve-config.yaml"
 
+os.environ["ILAB_MAX_STABLE_VRAM_WAIT"] = "0"
```

Comment:
Is it needed just for these test cases? I think you should isolate the effect then, by mocking it and reverting back after tests. Otherwise you potentially pollute environment for other test cases that run in the same process.

```
@mock.patch.dict(os.environ, {"ILAB_MAX_STABLE_VRAM_WAIT": "0"})
def test_max_stable_vram_wait():
    ...
```

Thoughts?
```

Reply:
We would want it for any unit test.  I was tempted to add it to init but figured that wouldn't be loved either.  The mock better, but I set it more globally to avoid anyone introducing 30s delays and not realizing it.

Reply:
Ah I think I see why you need to set it to 0, sorry missed your comment. This is to speed up tests. It makes sense but why not just pass it to pytest from tox env? There's `setenv` directive for this. The location of this magic setting is a bit hidden (why this and not other test module?)

Reply:
Moved to setenv.  Thanks for the suggestion!